awswrangler/distributed/_distributed.py

```diff
@@ -115,7 +119,11 @@ def initialize_ray(
                 "object_store_memory": object_store_memory,
                 "_redis_password": redis_password,
                 "_memory": object_store_memory,
+                "runtime_env": {
```

Comment:
@jaidisido Simply setting the env var at the top-level in CodeBuild doesn't work. Suspicion is the `runtime_env` variables don't get passed down from env vars in the workspace.